caf-logging-common/src/main/java/com/github/cafapi/logging/common/LogMessageValidator.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2019 Micro Focus or one of its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.github.cafapi.logging.common;
+
+import java.util.regex.Pattern;
+
+public final class LogMessageValidator
+{
+    public static final String PROCESS_ID_FORMAT = "#%s.%03d";
+    public static final Pattern DISALLOWED_CHARACTERS;
```

Comment:
Rather than making this public, why not have a `removeDisallowCharacters()` or `sanitize()` method to encapsulate the use of the regular expression.

Reply:
Added method to use regex